download.dd

```diff
@@ -238,9 +236,8 @@ Macros:
     SBTN=$(SPANC sig_btn,$(BTN $1,$+)<br>$(BTN $1.sig,sig))
     BTN=<a href="$1" class="btn">$+</a>
     H3I=<h3 class="download">$0</h3>
-    DLSITE=https://s3.us-west-2.amazonaws.com/downloads.dlang.org/releases/2.x/$(DMDV2)/$0
-    B_DLSITE=https://s3.us-west-2.amazonaws.com/downloads.dlang.org/pre-releases/2.x/$(B_DMDV2)/$0
-    N_DLSITE=https://s3.us-west-2.amazonaws.com/downloads.dlang.org/dmd-master/2.x/$0
```

Comment:
Not sure what this means, but should N_DLSITE be included too?

Reply:
See the commit message, I  removed it as it is unused. And it means "Nightly Download site".